spec.html

```diff
@@ -26906,13 +26906,12 @@ <h1>Number.prototype.toPrecision ( _precision_ )</h1>
       <emu-clause id="sec-number.prototype.tostring">
         <h1>Number.prototype.toString ( [ _radix_ ] )</h1>
         <emu-note>
-          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is not present or is *undefined* the Number 10 is used as the value of _radix_.</p>
+          <p>The optional _radix_ should be an integer value in the inclusive range 2 to 36. If _radix_ is *undefined*, 10 is assumed.</p>
```

Comment:
I don't like the "assumed" here. "the Number 10 is used as the value of _radix_" was fine IMO.

Reply:
Resolved in 4178c0d.